packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -84,22 +96,51 @@ export const useFeedLayout = ({
 }: UseFeedLayoutProps = {}): UseFeedLayoutReturn => {
   const isLaptop = useViewSize(ViewSize.Laptop);
   const { feedName } = useActiveFeedNameContext();
-  const shouldUseMobileFeedLayout = feedRelated
-    ? checkShouldUseMobileFeedLayout(isLaptop, feedName)
-    : !isLaptop;
+  const { insaneMode: listMode } = useContext(SettingsContext);
+  const { value: feedListLayoutExperiment } = useConditionalFeature({
+    feature: feature.feedListLayout,
+    shouldEvaluate: listMode && isLaptop,
+  });
+
+  const isListModeV1 =
+    feedListLayoutExperiment === FeedListLayoutExperiment.V1 && listMode;
+
+  const shouldUseListFeedLayoutOnProfilePages = UserProfileFeedPages.has(
+    feedName as UserProfileFeedType,
+  );
+
+  const isFeedIncludedInListLayout = FeedLayoutMobileFeedPages.has(
+    feedName as FeedPagesWithMobileLayoutType,
+  );
+
+  const shouldUseListFeedLayoutOnMobileTablet =
+    !isNullOrUndefined(isLaptop) && !isLaptop && isFeedIncludedInListLayout;
```

Comment:
Weird feels as the `!isLaptop` would already be enough here.

Reply:
Not enough because it's taking the contrary value of what it should take.